features/step_definitions/branch_steps.rb

```diff
@@ -33,7 +33,7 @@
   in_repository :coworker do
     create_branch branch_name
     if relation
-      commit_to_branch = relation == 'behind' ? 'main' : branch_name
+      commit_to_branch = relation == "behind" ? 'main' : branch_name
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.